<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="GUS-LV" failures="1" tests="8" name="Pruebas Registro e Inicio de Sesión" time="30.318" errors="0" timestamp="2022-05-24T00:55:49 CST">
  <testcase classname="TestCases.Auth" name="TestCaseOptionRegister" time="1.824"/>
  <testcase classname="TestCases.Auth" name="TestCaseRegisterWithoutPrivacyPolicy" time="2.626"/>
  <testcase classname="TestCases.Auth" name="TestCaseRegisterSuccess" time="5.286"/>
  <testcase classname="TestCases.Auth" name="TestCaseRegisterRepeatedEmail" time="2.644"/>
  <testcase classname="TestCases.Auth" name="TestCaseOptionLogin" time="0.746"/>
  <testcase classname="TestCases.Auth" name="TestCaseLoginIncorrect" time="1.638"/>
  <testcase classname="TestCases.Auth" name="TestCaseLoginSuccess" time="0.005">
    <failure type="java.lang.NullPointerException" message="Cannot invoke &amp;quot;org.json.simple.JSONObject.get(Object)&amp;quot; because &amp;quot;data&amp;quot; is null">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "org.json.simple.JSONObject.get(Object)" because "data" is null
at TestCases.Auth.readDataForLogin(Auth.java:201)
at TestCases.Auth.TestCaseLoginSuccess(Auth.java:133)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TestCaseLoginSuccess -->
  <testcase classname="TestCases.Auth" name="TestCaseLogout" time="3.923"/>
</testsuite> <!-- Pruebas Registro e Inicio de Sesión -->
